app/delivery/send_to_providers.py

```diff
@@ -231,27 +233,30 @@ def send_email_to_provider(notification: Notification):
             check_file_url(personalisation_data[key]["document"], notification.id)
             sending_method = personalisation_data[key]["document"].get("sending_method")
             direct_file_url = personalisation_data[key]["document"]["direct_file_url"]
+            filename = personalisation_data[key]["document"].get("filename")
+            mime_type = personalisation_data[key]["document"].get("mime_type")
             document_id = personalisation_data[key]["document"]["id"]
             scan_verdict_response = document_download_client.check_scan_verdict(service.id, document_id, sending_method)
             check_for_malware_errors(scan_verdict_response.status_code, notification)
             current_app.logger.info(f"scan_verdict for document_id {document_id} is {scan_verdict_response.json()}")
             if sending_method == "attach":
                 try:
-                    req = urllib.request.Request(direct_file_url)
-                    with urllib.request.urlopen(req) as response:
-                        buffer = response.read()
-                        filename = personalisation_data[key]["document"].get("filename")
-                        mime_type = personalisation_data[key]["document"].get("mime_type")
-                        attachments.append(
-                            {
-                                "name": filename,
-                                "data": buffer,
-                                "mime_type": mime_type,
-                            }
-                        )
+                    logging.getLogger("urllib3").setLevel(logging.DEBUG)
```

Comment:
why is this logging level debug? can it be warning?

Reply:
Good catch, this should be removed. I was using it to debug which calls `urllib3` was making internally for mocking in tests and forgot to remove it.